nautobot/core/api/utils.py

```diff
@@ -167,28 +167,31 @@ def is_api_request(request):
     return request.path_info.startswith(api_path)
 
 
-def get_view_name(view, suffix=None):
+def get_view_name(view):
```

Comment:
Do current versions of DRF no longer expect a `suffix` optional argument for this API?

Reply:
Yeah, it was changed ~6 years ago in [DRF #5605](https://github.com/encode/django-rest-framework/pull/5605). It changed it to looking for it as an attribute of the view itself.

Looks like it changed in [3.9.0](https://www.django-rest-framework.org/community/release-notes/#390) (18th October 2018).

I did a search against the nautobot org and didn't see any references to using the suffix (or the function itself). The only references were `APIRootView`s setting the app name which doesn't matter to this function.

Reply:
Thanks for confirming. That aligns with your previous observations about the age of this code. :-)